src/index.rst

```diff
@@ -178,6 +178,7 @@ LLNL-CODE-741217
    :maxdepth: 1
    :caption: Component User Guides
 
+   Core (Widely useful utilities) <axom/core/docs/sphinx/index>
```

Comment:
New name for Core? WUU? =)

Reply:
heh.  I couldn't think of a better phrase to put in the parens.